packages/module-federation/migrations.json

```diff
@@ -1 +1,25 @@
-{}
+{
+  "packageJsonUpdates": {
+    "20.4.0": {
+      "version": "20.4.0-beta.0",
+      "packages": {
+        "@module-federation/enhanced": {
+          "version": "0.8.8",
+          "alwaysAddToPackageJson": false
+        },
+        "@module-federation/runtime": {
+          "version": "0.8.8",
+          "alwaysAddToPackageJson": false
+        },
+        "@module-federation/sdk": {
+          "version": "0.8.8",
+          "alwaysAddToPackageJson": false
+        },
+        "@module-federation/node": {
+          "version": "2.6.21",
+          "alwaysAddToPackageJson": false
+        }
```

Comment:
Do they need to be pinned versions?

The migration version for `@module-federation/node` would pin it, while new projects would get the range `^2.6.21` based on the constant in the `versions.ts` file.

The other three packages match their constant but don't match the package.json dependencies where they are set to the range `^0.8.8`.

Reply:
No, they can be a range, I'll update